frappe/translations/de.csv

```diff
@@ -1480,7 +1480,8 @@ Linked with {0},Verknüpft mit {0},
 Links,Verknüpfungen,
 List,Listenansicht,
 List Filter,Listenfilter,
-List View Setting,List View Setting,
+List View,Listenansicht,
+List View Setting,Einstellungen zu Listenansicht,
```

Comment:
`List view Setting` is singular but `Einstellungen zu Listenansicht` seems to be plural

![image](https://user-images.githubusercontent.com/28212972/152394604-5e087f1c-001b-43a0-b99e-790b06ee7605.png)

![image](https://user-images.githubusercontent.com/28212972/152394659-a5d328ce-b9b2-42ca-8056-d056868c645b.png)

I don't really know german but since this translation is getting directly added to the codebase, it would be nice to add an explaination

Reply:
In other words: The first would be "Settings regarding List view", the second would be something like "Setup of List view".

Will that help?

I'd say the second example "Einstellung der Listenansicht" is easier to read.